src/open_inwoner/components/templates/components/Header/WarningHeader.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n link_tags static %}
+
+<style>
+	.warning-header {
+		background-color: {{ warning_banner_background_color }};
```

Comment:
I don't see the warning_banner_font_color option used here?

Also, as we have CSP headers restricting inline styles this won't work out of the box on non-dev environments.

See the use of request.csp_nonce in templates/master.html, this will be the only way your style can be picked up.